code/ui/components/src/Zoom/browserSupportsCssZoom.ts

```diff
@@ -2,8 +2,12 @@ import { global } from '@storybook/global';
 
 export function browserSupportsCssZoom(): boolean {
   try {
-    // @ts-expect-error (we're testing for browser support)
-    return global.document.implementation.createHTMLDocument('').body.style.zoom !== undefined;
+    const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
```

Comment:
I think a code-comment here would be great explaining why this odd code is here, and when it'd be safe to remove, assuming the big will eventually be fixed in safari?

Can we have a link to the issue in the safari bugtracker?

This Regex mentions chrome and android, why?